server.js

```diff
@@ -0,0 +1,26 @@
+require('dotenv').config();
+
+const { createApp } = require('./app')
+const appDataSource = require('./api/models/dataSource')
+
+const startServer = async() => {
+    const app = createApp();
+    const PORT = process.env.PORT
+    
+        await appDataSource.initialize()
+            .then(() => {
+            console.log("Data Source has been initialized")
+        })
+            .catch((err) => {
+            console.error('Failed to initialize Data Source', err)
+            AppData.destroy();
```

Comment:
distory 할 instance가 잘못 지정되어 있는것 같습니다. appDataSource라는 변수명으로 만들어 할당해 주었기 때문에 .destory() method가 실행 될 주체도 마찬가지로 맞춰줘야합니다!
```suggestion
            appDataSource.destroy();
```

Reply:
해당  부분 다시 수정하여 push 하였습니다~!